interfaces/HO-Portal/src/app/user/user.page.html

```diff
@@ -39,21 +43,50 @@ <h2>{{ 'page.user.change-password.title'|translate }}</h2>
             >
               <div class="input-item ion-margin-bottom ion-padding-bottom">
                 <label color="medium" class="input-item--label auth-label">{{ 'page.user.change-password.new-password'|translate }}</label>
-                <ion-input
-                  type="password"
-                  autocomplete="new-password"
-                  autocapitalize="off"
-                  clearInput="true"
-                  clearOnEdit="false"
-                  required
-                  [minlength]="8"
-                  name="new-password"
-                  [(ngModel)]="newPassword"
-                  ngDefaultControl
-                  class="ion-margin-vertical">
-                </ion-input>
+                <div class="input-item--content">
+                  <ion-input
+                    type="password"
+                    autocomlete="off"
```

Comment:
Why change this?
It can trigger users' password-managers to insert/prompt for the new password.

Reply:
It does it anyway

![image](https://user-images.githubusercontent.com/63000678/146371620-07a2ea16-949c-488e-84b2-0ce833aa9e6f.png)

But I can change it back